File: @LV1/1.3/ladder/ladder.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var h, v float64
	fmt.Scanf("%f %f", &h, &v)
	ans := h / math.Sin(v/180*math.Pi)
	fmt.Println(math.Ceil(ans))
}

File: @LV1/1.3/ladder/ladder.rs
use std::f64::consts::PI;

fn main() {
    let stdin = std::io::stdin();
    let mut buff = String::new();
    stdin.read_line(&mut buff).unwrap();
    let num: Vec<f64> = buff
        .split_whitespace()
        .map(|l| l.parse().unwrap())
        .collect();

    let ans = num[0] / (num[1] / 180.0 * PI).sin();
    println!("{}", ans.ceil());
}
